nbs/models.tsmixerx.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        <code>num_workers_loader</code>: int=os.cpu_count(), workers to be used by <code>TimeSeriesDataLoader</code>.

</pre><p>missing new optimizer parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Are that these only?</p><p> </p><p><code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;optimizer = None,</code></p><p><code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;optimizer_kwargs = None,</code></p><p> </p>

Reply:
Added these two to the input args in both TSMixer and TSMixerx